src/index.html

```diff
@@ -6,13 +6,74 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
+    <title>Product product-cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__product-image"
+        src="./images/imac.jpeg"
+        alt="apple-imac-27"
+      />
+
+      <div class="product-card__product-details">
+        <h3 class="product-card__product-details-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="product-card__product-details-description">
+          Product code: 195434
+        </p>
+      </div>
+
+      <div class="product-card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="product-card__rating-reviews">Reviews: 5</div>
```

Comment:
You need to use the semantic tag for text, fix it everywhere
```suggestion
        <p class="product-card__rating-reviews">Reviews: 5</p>
```